ow_plexil/src/plans/FaultDependenciesModel.xml

```diff
@@ -0,0 +1,95 @@
+<System>
+  <Subsystems>
+
+    <Subsystem Name="Power">
+      <Faults>
+        <Fault Name="LOW_STATE_OF_CHARGE">
+          <Impacts Name="Power" Type="Subsystem"/>
+        </Fault>
+        <Fault Name="INSTANTANEOUS_CAPACITY_LOSS">
+        </Fault>
+        <Fault Name="THERMAL_FAULT">
+          <Impacts Name="Power" Type="Subsystem"/>
+        </Fault>
+      </Faults>
+
+      <Impacts Name="Arm" Type="Subsystem"/>  
+      <Impacts Name="Antenna" Type="Subsystem"/>  
+      <Impacts Name="Camera" Type="Subsystem"/>  
```

Comment:
In reality, only the low SOC and thermal faults impact these systems.  You can leave them here for illustrative purposes or move them up, as you like.

Reply:
That is a good point. I think I will leave them for now, as it is a good example for subsystem impacting another subsystem.